app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -5,6 +5,18 @@ def self.display_name(number = 1)
     n_('Repository (Embedded Workflows)', 'Repositories (Embedded Workflows)', number)
   end
 
+  def self.seed
+    manager = ManageIQ::Providers::Workflows::AutomationManager.in_my_region.first
+    return if manager.nil?
+
+    manager.configuration_script_sources.find_or_create_by!(:type => name, :name => "Embedded Workflows")
```

Comment:
Not sure about the name here.  I think I'd prefer `ManageIQ` as the name similar to how we named the built-in automate domain as `ManageIQ`, but I'm not sure.  Productization-wise it's a little weird.

Reply:
Sounds good, how to we productize the builtin automate domain?  We could use a Setting and then override that